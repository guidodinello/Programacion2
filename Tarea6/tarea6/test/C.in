apilar 5
encolar 2
encolar 8
encolar 1 
estaVacioAvl
insertarEnAvl 10
insertarEnAvl 12
insertarEnAvl 8
insertarEnAvl 6
insertarEnAvl 4
unionDeConjuntos 2 3 6
diferenciaDeConjuntos 4 2 3 4 6
insertarEnCP 4 4.0
insertarEnCP 9 9.0
insertarEnCP 3 3.0
desapilar
estaVaciaPila
desencolar
desencolar
frente
raizAvl
alturaDeAvl
enOrdenAvl
cardinalidad
perteneceAConjunto 6
estaVacioConjunto
iteradorDeConjunto
existeAsociacionEnMap 5
prioridad 3
actualizarEnCP 9 30.0
eliminarPrioritario
prioritario
Fin
