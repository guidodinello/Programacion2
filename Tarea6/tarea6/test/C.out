1>Apilado.
2>Encolado.
3>Encolado.
4>Encolado.
5>Vacio.
6>Insertado.
7>Insertado.
8>Insertado.
9>Insertado.
10>Insertado.
11>Union.
12>Diferencia.
13>Se insertó el elemento.
14>Se insertó el elemento.
15>Se insertó el elemento.
16>Desapilado.
17>p está vacia.
18>Desencolado.
19>Desencolado.
20>Binario vacío.
21>10
22>Altura de avl: 3.
23>4 6 8 10 12 
24>0.
25>No pertenece.
26>Vacio.
27>No hay elementos en el conjunto.
28>No existe asociacion.
29>3.00.
30>Se actualizó el valor.
31>Eliminado.
32>4
33>Fin.
